data/flux_stdlib_frontmatter.yml

```diff
@@ -920,11 +920,46 @@
   aliases:
     - /influxdb/v2/reference/flux/stdlib/experimental/mqtt/
     - /influxdb/cloud/reference/flux/stdlib/experimental/mqtt/
+  prepend:
+    block: warn
+    content: |
+      #### Not supported in InfluxDB OSS 2.7+
+
+      The `experimental/mqtt` package is not supported in InfluxDB 2.7+, but
+      is supported in InfluxDB Cloud (TSM). The package is still available to
+      import in OSS, but functions will not successfully publish to an MQTT broker.
+  exclude_from:
+    oss: ^2\.([7-9]|1[0-9])
+    nightly: true
 
 /flux/v0/stdlib/experimental/mqtt/to.md: |
   aliases:
     - /influxdb/v2/reference/flux/stdlib/experimental/mqtt/to/
     - /influxdb/cloud/reference/flux/stdlib/experimental/mqtt/to/
+  prepend:
+    block: warn
+    content: |
+      #### Not supported in InfluxDB OSS 2.7+
+
+      The `experimental/mqtt` package is not supported in InfluxDB 2.7+, but
```

Comment:
I think it would be best to say `InfluxDB OSS 2.x` or whatever we use to designate all the OSS 2 versions.  Say `2,7/=` implies it is supported in earlier 2,x versions.

Reply:
Was support for it removed from all previous v2 versions?

Reply:
I don't think it was ever supported on 2.X OSS.

Reply:
Ok, then that likely means it was never supported in OSS v1 as well?

Reply:
I doubt it, but have never tested it.  I suspect that it was only usable in C2.

Reply:
@davidby-influx I just pushed a change that removes support from all InfluxDB versions except InfluxDB Cloud 2 (TSM). I also updated the text in the warning.

Side question--does this package rely on dependencies outside of the Flux runtime? What does this work in Cloud, but not anything else?